app.py

```
@@ -0,0 +1,6870 @@
+                    )
+                }
+            except:
+                files = {
+                    "jd_file": open(
+                        base_dir + "/" + "media/uploaded_jds/" + filename, "rb"
+                    )
+                }
+            result = requests.post(url, headers=headers, files=files)
+            logger.debug("JD Parser API response " + str(result))
+            parser_output = result.json()
+            try:
+                with open(
+                    base_dir + "/" + "media/jd_output/" + filename + ".json", "w"
+                ) as fp:
+                    json.dump(parser_output, fp)
+            except:
+                with open(
+                    os.getcwd() + "/" + "media/jd_output/" + filename + ".json", "w"
+                ) as fp:
+                    json.dump(parser_output, fp)
+            qual_name = []
+            role_and_res = []
+            try:
+                job_title = ", ".join(parser_output["job_title"])
+            except:
+                job_title = ""
+            try:
+                qual = parser_output["edu_qualification"]
+                import re
+
+                qual_list = re.split(", |_|-|!|\+", qual[0])
+                for i in qual_list:
+                    if i.lower().strip() in settings.ug:
+                        qual_name.append({"qual": "Bachelors"})
+                    if i.lower().strip() in settings.pg:
+                        qual_name.append({"qual": "Masters"})
+                    if i.lower().strip() in settings.phd:
+                        qual_name.append({"qual": "Doctorate"})
+            except KeyError:
+                pass
+            # [dict(t) for t in {tuple(d.items()) for d in l}]
+            # result = [dict(tupleized) for tupleized in set(tuple(item.items()) for item in l)]
+            try:
+                add_in = parser_output["Additional Information"]
+
+            except:
```

Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```

Similar issue at line numbers 4202, 4211, 4217, 4227, and 4237.